README.md

```diff
@@ -0,0 +1,198 @@
+# Wagtail Content Import
+
+A module for Wagtail that provides functionality for importing page content from third-party sources.
+Currently supports importing Google Docs into a StreamField.
+
+## Getting Started
+
+### Requirements:
+* Django 2.2
+* Wagtail 2.2
+* Python 3
+
+### To install:
+ 1. Run `python3 pip install wagtail-content-import`.
+ 2. Add `'wagtail_content_import'` and `'wagtail_content_import.pickers.google'` to `INSTALLED_APPS`.
+ 
+### To set up:
+ Wagtail Google Docs integration relies on Google APIs, which you will first need to enable for your project:
+
+1. Navigate to the [Google API Library](https://console.developers.google.com/apis/library). Select a project for your Wagtail site, or create a new one now.
+
+2. Find and enable the [Google Docs](https://console.developers.google.com/apis/library/docs.googleapis.com) and [Google Drive](https://console.developers.google.com/apis/library/drive.googleapis.com) APIs.
+    
+3. Find and enable the [Google Picker](https://console.developers.google.com/apis/api/picker.googleapis.com) API, and copy its API key to the setting `GOOGLE_PICKER_API_KEY`.
+
+4. Open the [Credentials](https://console.developers.google.com/apis/credentials) page in the API Console.
+
+5. Select `Create credentials`, then `OAuth client ID`
+
+6. If you haven't already configured the consent screen, you will need to configure this now.
+
+    1. Under `Scopes for Google APIs`, click `Add scope`.
+
+    2. Add `../auth/documents.readonly` and `..auth/drive.readonly` scopes.
+
+        Note: adding these sensitive scopes means that you will need to submit your project for verification by Google to
+        avoid user caps and warning pages during use.
+        
+    3. Add your domain to `Authorised domains`.
+
+ 7. For `Application type`, choose `Web application`
+
+ 8. Under `Authorised JavaScript origins`, add your domain.
+
+ 9. On the Credentials page, next to your Client ID, click the download item to download a JSON file of your client
+    secret.
+
+ 10. Copy the text from this file, and use it to set `GOOGLE_OAUTH_CLIENT_CONFIG`.
+
+## Basic Usage
+
+1. To enable Google Docs import for a Page model, it should inherit from `ContentImportMixin` 
+(`wagtail_content_import.models.ContentImportMixin`). By default, content will be imported into into a StreamField called body
+(see [Changing Import Fields](#Changing-Import-Fields) for how to change this).
+
+2. You'll then need to create a Mapper, which maps the parsed document into your StreamField blocks. Create a class deriving from `wagtail_content_import.mappers.streamfield.StreamFieldMapper`:
+
+    ```
```

Comment:
worth adding "python` after the 3 opening backticks - will add nice syntax highlighting

Reply:
Thanks for the tip! I've added that now.